test/constants.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+
+// https://russelldavies.github.io/jwk-creator/
```

Comment:
what is this?

Reply:
just a tool I found handy for creating jwk's for tests, I went through a few packages that didn't work, so thought I'd leave it there for other devs, but I'll remove it if it's messy

Reply:
yes let's not reference it here. a better place is the docs but it's somewhere in another package's docs so we don't strictly need it here. remove